editor/action_map_editor.cpp

```diff
@@ -58,10 +58,16 @@ void ActionMapEditor::_event_config_confirmed() {
 	Array events = new_action["events"].duplicate();
 
 	if (current_action_event_index == -1) {
-		// Add new event
+		// Add new event if not already bound.
+		for (const Variant &event : events) {
+			Ref<InputEvent> event_to_check = Ref<InputEvent>(event);
```

Comment:
```suggestion
		for (Ref<InputEvent> event_to_check events) {
```

Reply:
Thanks, agreed that this is a bit more succinct. Also made the reference const just because we're not mutating these events while iterating over them.